src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```diff
@@ -24,13 +24,13 @@
       See: https://github.com/dotnet/sdk/blob/955c0fc7b06e2fa34bacd076ed39f61e4fb61716/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets#L16
     -->
     <_GetChildProjectCopyToPublishDirectoryItems>false</_GetChildProjectCopyToPublishDirectoryItems>
-    <!-- Define a $(_AndroidNativeAot) property, as the logic detecting NativeAOT may change in the future -->
-    <_AndroidNativeAot Condition=" '$(PublishAot)' == 'true' ">true</_AndroidNativeAot>
-    <_AndroidNativeAot Condition=" '$(_AndroidNativeAot)' == '' ">false</_AndroidNativeAot>
-    <UseMonoRuntime Condition=" '$(_AndroidNativeAot)' == 'true' and '$(UseMonoRuntime)' == '' ">false</UseMonoRuntime>
+    <UseMonoRuntime Condition=" '$(PublishAot)' == 'true' and '$(UseMonoRuntime)' == '' ">false</UseMonoRuntime>
     <UseMonoRuntime Condition=" '$(UseMonoRuntime)' == '' ">true</UseMonoRuntime>
+    <_AndroidRuntime Condition=" '$(PublishAot)' == 'true' and '$(UseMonoRuntime)' != 'true' ">NativeAOT</_AndroidRuntime>
+    <_AndroidRuntime Condition=" '$(PublishAot)' != 'true' and '$(UseMonoRuntime)' != 'true' ">CoreCLR</_AndroidRuntime>
+    <_AndroidRuntime Condition=" '$(_AndroidRuntime)' == '' ">MonoVM</_AndroidRuntime>
```

Comment:
I don't quite understand the desired semantics; when would `$(UseMonoRuntime)` be set to false?  When would CoreCLR be used?  Or is it that CoreCLR support is still in-progress via #9572, and thus it *currently* isn't possible for `$(_AndroidRuntime)` to be CoreCLR, and #9572 will change that?  Or is the expectation that in the future, to use CoreCLR then devs must explicitly set `$(UseMonoRuntime)`=false in their .csproj?

Reply:
Right now, the best option we have to align with xamarin/xamarin-macios is that you would set `UseMonoRuntime=false` in your `.csproj` file.